docs/en/registry.rst

```diff
@@ -765,4 +872,60 @@ The registry components are interconnected and work together to support the comp
+
+Credential Issuance
+^^^^^^^^^^^^^^^^^^^^^^^^
+
+This journey defines how a **Credential Issuer (CI)** uses the Registry Infrastructure to prepare and issue a compliant Digital Credential.
+
+1.  **Identifying Requirements**: The CI consults the **Digital Credentials Catalog** for the technical requirements of the credential type to be issued (e.g., `max_validity_days`, `min_loa`).
+2.  **Schema and Claim Resolution**:
+    * The CI consults the **Schema Registry** to retrieve the technical specification of the format and schema (e.g., JSON Schema for SD-JWT) required by the Catalog, ensuring validity and integrity via the hash (`schema_uri#integrity`).
+    * The CI accesses the **Claims Registry** to retrieve the standardized semantic definitions and data formats (data types) of the necessary attributes (claims).
+3.  **Authentic Data Retrieval**:
+    * The CI consults the **Authentic Source (AS) Registry** to identify the authorized **Authentic Source** (AS) for the required dataset. The AS Registry provides the AS's ``entity_id`` and the technical details of the interface (`integration_endpoint`, `integration_method`).
+    * The CI consults the AS endpoint specification to implement the integration needed to retrieve the user data required to populate the  Digital Credential.
+4.  **Credential Issuance**: The CI uses the retrieved data, validated schemas, and specified formats to generate and sign the Digital Credential in the correct format (e.g., SD-JWT or mDOC).
+
+Credential Presentation and Verification
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+This journey describes how a **Wallet Instance** and a **Relying Party (RP)** interact with the Registry Infrastructure when a Digital Credential needs to be presented by a User.
+
+1.  **Wallet Authorization and Selection**:
+    * The Wallet receives a Presentation Request from the RP, verifies the validity of the request comparing the requested *claims* with the *Authorization Policies* related to the RP (via the **Taxonomy** definitions).
+    * The Wallet consults the **Digital Credentials Catalog** to verify the *Domains* and *Purposes* associated with the credential types it holds, evaluating which credentials are suitable for the request.
+    * The Wallet verifies if the required attributes (claims) are available and authorized for disclosure based on the request policy (**Credential-Specific** or **Credential-Agnostic** scenarios).
+    * The User authorizes the release of the selected, selectively disclosed attributes. The Wallet then packages and presents the Digital Credential to the RP.
+
+2.  **Discovery and Integrity**:
+    * The RP receives the Digital Credential from the User.
+    * The RP consults the **Federation Registry** via the Trust Anchor's endpoint (`federation_resolve`, `federation_trust_mark_status`) to verify the **cryptographic trust** (Trust Mark) of the Issuer and Wallet Provider.
```

Comment:
```suggestion
    * The RP consults the **Federation Registry** via the Trust Anchor's endpoint (`federation_resolve`, `federation_trust_mark_status`) to verify the **cryptographic trust** (Trust Mark) of the Issuer and Wallet Provider.
```
we should point to the trust section, where we describe all the trust discovery patterns by use case